classes/PodsAPI.php

```diff
@@ -1753,12 +1760,28 @@ public function save_pod( $params, $sanitized = false, $db = true ) {
 					return pods_error( sprintf( __( 'Pod %s already exists', 'pods' ), $params->name ), $this );
 				}
 			}
-		} elseif (
-			in_array( $params->name, pods_reserved_keywords(), true )
-			&& in_array( pods_v( 'type', $params ), array( 'post_type', 'taxonomy' ), true )
-		) {
-			return pods_error( sprintf( __( '%s is reserved for internal WordPress or Pods usage, please try a different name', 'pods' ), $params->name ), $this );
 		} else {
+			// New pod (create).
+
+			if (
+				in_array( $params->name, pods_reserved_keywords(), true )
+				&& in_array( pods_v( 'type', $params ), array( 'post_type', 'taxonomy' ), true )
+			) {
+				$valid_name = false;
+
+				if ( $extend ) {
```

Comment:
```suggestion
				if ( ! $extend ) {
```

@JoryHogeveen It seems like we only want to validate on creating new ones, extending existing ones is not under our control and the plugin/theme that created the post type / taxonomy would have to mitigate their own risks.

Reply:
Correct, but this code works the other way around.
If rejects all reserved keywords (see line 1767).
When if finds a reserved keyword THEN it checks if `$extend` is true. If it's true then existing CPT/Tax are allowed.

Reply:
I'll add inline documentation